### ISSUE TITLE
Improve integration with Trello

### ISSUE THREAD

**Issue by mikolak-net** [mikolak-net/pomisos#2]
- add support for checklists
 - preserve ordering when syncing

**Comment by scalway** [comment]
in my opinion ordering should be same as in trello column. Currently it is impossible to change order and this is reasonable behavior.

**Comment by mikolak-net** [comment]
@scalway : yup, that's the second point. And the column ordering itself would be:
Doing
ToDo
Done

If you have other suggestions, I'm all ears. And of course, PRs are welcome ;).
